product/r7insight.js

```diff
@@ -392,29 +396,30 @@
         createLogStream: _createLogStream,
         to: _getLogger,
         destroy: _destroyLogStream,
-        log: function() {
+        log: function () {
             for (var k in loggers)
                 loggers[k].log.apply(this, arguments);
         },
-        warn: function() {
+        warn: function () {
             for (var k in loggers)
                 loggers[k].warn.apply(this, arguments);
         },
-        error: function() {
+        error: function () {
             for (var k in loggers)
                 loggers[k].error.apply(this, arguments);
         },
-        info: function() {
+        info: function () {
             for (var k in loggers)
                 loggers[k].info.apply(this, arguments);
         }
     };
 
     function validate_region(region) {
-        var allowed_regions = ['eu', 'us', 'us2', 'us3', 'ca', 'au', 'ap'];
+        var allowed_regions = ['eu', 'us', 'us2', 'us3', 'ca', 'au', 'ap','custom'];
```

Comment:
personally I would prefer a flag to ignore region instead of `custom` value added

Reply:
I left maintainer commits turned on.